packages/common/src/services/audius-backend/solana.ts

```diff
@@ -405,6 +405,39 @@ export const purchaseContentWithPaymentRouter = async (
   return tx
 }
 
+export type PurchaseContentWithPaymentRouterUserBankArgs = {
+  id: number
+  type: 'track'
+  splits: Record<string, number>
+  extraAmount?: number
+  blocknumber: number
+  recentBlockhash?: string
+  purchaserUserId: ID
+}
+
+export const purchaseContentWithPaymentRouterUserBank = async (
```

Comment:
Would be nice to have comments on the various purchase methods here. The names are just a little cryptic if you're not in the middle of working on this feature :-)